Add disclaimer, and fix links (#5)
model-checking/verify-rust-std: README.md

To help you get started in the contest, Amazon’s [Kani-rust-verifier team](https://github.com/model-checking/kani) has created mechanisms and tools to help participants verify the standard library. The Kani team has also created some initial contracts and proofs (To be filled and linked here later) to help you get started as a participant.

NOTE: This work is not official, affiliated, or endorsed by the Rust project or Rust Foundation.
* * *

## Contest Details

Here are some details for the contest

> 1. This repository will contain templates for [Issues](https://github.com/model-checking/verify-rust-std/issues), [pull requests](https://github.com/model-checking/verify-rust-std/pulls) etc. that will be used to create new challenges for verifying the Rust standard library.

Reviewer: ```suggestion
1. This repository will contain templates for [Issues](https://github.com/model-checking/verify-rust-std/issues), and [pull requests](https://github.com/model-checking/verify-rust-std/pulls), that will be used to create new challenges for verifying the Rust standard library.
```

## Contact

> For questions, suggestions or feedback, feel free to open an [issue on the Kani page](https://github.com/model-checking/kani/issues) with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).

Reviewer: Shouldn't they just create an issue in this repository instead?

Author: I thought we wanted to save issues in this repository for challenges, and we can use a discussions tab to drive discussions about these challenges, and meta-level issues with the contest itself should be directed to Kani. lmk if you disagree.

> For questions, suggestions or feedback, feel free to open an [issue on the Kani page](https://github.com/model-checking/verify-rust-std/issues) with the tag `stdlib-contest`  or contact us directly at [kani-developers@amazon.com](mailto:kani-developers@amazon.com).

Reviewer: ```suggestion
For questions, suggestions or feedback, feel free to open an [issue here](https://github.com/model-checking/verify-rust-std/issues).
```
